Source/NSString.m

```diff
@@ -6765,3 +6775,309 @@ - (void) setString: (NSString*)aString
 
 @end
 
+/**
+ * An item in a .stringsdict file.
+ * https://developer.apple.com/documentation/xcode/localizing-strings-that-contain-plurals
+ */
+@implementation GSLocalizedString
+
+- (id)initWithDictionary:(NSDictionary *)dict defaultValue:(NSString *)value
+{
+  _dict = dict;
+  return [self initWithString:value];
+}
+
+/* Returns a format that's appropriate for the pluralization of the arguments in
+ * the current language */
+- (NSString *)getPluralizedFormat:(va_list)argList
+{
+  /* Sample dictionary:
+      NSStringLocalizedFormatKey=%#@x_hours@, %#@x_minutes@
+      x_hours=NSDictionary {
+	NSStringFormatSpecTypeKey=NSStringPluralRuleType
+	NSStringFormatValueTypeKey=ld
+	zero=0 hours
+	one=1 hour
+	other=%1$ld hours
+      }
+      x_minutes=NSDictionary {
+	NSStringFormatSpecTypeKey=NSStringPluralRuleType
+	NSStringFormatValueTypeKey=ld
+	zero=0 minutes
+	one=1 minute
+	other=%2$ld minutes
+      }
+   */
+
+  NSString *formatKey = _dict[@"NSStringLocalizedFormatKey"];
+  if (!formatKey)
+    {
+      return self;
+    }
+
+  static dispatch_once_t      onceToken;
+  static NSRegularExpression *regex = nil;
+
+  regex = [NSRegularExpression
+    regularExpressionWithPattern:@"%#@([^@]+)@"
+			 options:NSRegularExpressionCaseInsensitive
+			   error:nil];
```

Comment:
I'm guessing the dispatch_once_t is leftover code from caching at some point?
Can we remove it alongside the dispatch header includes? Also remove the static qualifier from the regex